lib/containers/common.pm

```diff
@@ -326,4 +328,23 @@ sub check_containers_connectivity {
     assert_script_run "$runtime kill $container_name";
 }
 
+sub switch_cgroup {
+    my ($self, $version) = @_;
+
+    my $setting = ($version == 2) ? 1 : 0;
```

Comment:
I think it would be better to switch from v1 to v2 by default. IMHO that is more user-friendly